WordPress-Theme/ruleset.xml

```diff
@@ -3,7 +3,7 @@
 	<!-- For more information: https://make.wordpress.org/themes/handbook/review/ -->
 	<description>Standards any Theme to be published on wordpress.org should comply with.</description>
 
-
+	<rule ref="WordPress.Functions.FunctionRestrictions"/>
```

Comment:
We'll be adding a new folder to the `WordPress/Sniffs` and `WordPress/Tests` directories names `Theme`. That folder will contain all sniffs (and it's sister the unit tests) which are very specific to the theme review process.

If you have a look at PR #10 where I've done something similar to what you're doing here, you can see what I mean and what that will look like.

Once the first custom sniff which will live in the `Theme` folder has been merged, the rule `<rule ref="WordPress-Theme"/>` will be added to the ` WordPress-Theme/ruleset.xml`. From then on, all sniffs contained within the `Theme` folder will automatically be registered and run when running PHPCS against the new `WordPress-Theme` standard.